src/tools/std20/views/test/iota.test.cpp

```diff
@@ -0,0 +1,145 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/iterator.hpp"
+#include "tools/std20/views.hpp"
+
+// other includes
+#include "tools/std20/algorithm.hpp"
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "iota_view" ) {
+
+  const std::vector< int > equal = { 1, 2, 3, 4, 5 };
+
+  GIVEN( "a value" ) {
+
+    WHEN( "creating an iota view" ) {
+
+      auto chunk = std20::ranges::iota_view< int, int >( 1 );
+      using Range = decltype( chunk );
+      using Iterator = nano::iterator_t< Range >;
+
+      THEN( "the transform_view satisfies the required concepts" ) {
```

Comment:
"transform_view" -> "iota_view"

Likewise with other THENs in this scenario

Reply:
same